src/main/scala/uk/gov/nationalarchives/notifications/messages/Messages.scala

```diff
@@ -24,18 +24,32 @@ trait Messages[T <: IncomingEvent, TContext] {
   def slack(incomingEvent: T, context: TContext): Option[SlackMessage]
 
   def sqs(incomingEvent: T, context: TContext): Option[SqsMessageDetails]
+
+  def sns(incomingEvent: T, context: TContext): Option[SnsMessageDetails]
```

Comment:
This is just a suggestion so feel free to ignore it but you could give this a default value of `None` here and they you wouldn't have to implement it for all of the `Messages` instances. 

I know the existing fields don't have defaults and there's a lot of lines like `override def sqs(incomingEvent: ScanEvent, context: ImageScanReport): Option[SqsMessageDetails] = Option.empty` but we can tidy that up later. 

I don't think I've explained this very well so let me know if this doesn't make sense.

Reply:
Thanks this makes a lot of sense. I've defaulted it to `None` and removed all the empty overrides from the messages